snomed/com.b2international.snowowl.validation.snomed/src/main/resources/scripts/rule_mrcm_constraint.groovy

```diff
@@ -50,14 +70,18 @@ searcher.search(mrcmRangeMemberQuery).each { hit ->
 	String typeId = hit[1];
 	String rangeConstraint = hit[2];
 	
-	if (allowedRanges.containsKey(typeId)) {
-		if (allPrecoordinatedContent.equals(contentType)) {
-			allowedRanges.put(typeId, rangeConstraint);
+	if (rangeConstraint.startsWith(integerTypeRangePrefix) || rangeConstraint.startsWith(decimalTypeRangePrefix)) {
+		//Do nothing, skip concrete value type range validation for now
+	} else {
+		if (allowedRanges.containsKey(typeId)) {
+			if (allPrecoordinatedContent.equals(contentType)) {
+				allowedRanges.put(typeId, rangeConstraint);
+			} else {
+				//Do nothing, the already mapped member should be chosen over this
+			}
```

Comment:
Same concern as above wrt. "all pre-coordinated content" mode and overwriting the Map entry.